docs/docs/guides/automate/sensors/testing-run-status-sensors.md

```diff
@@ -4,6 +4,10 @@ sidebar_position: 600
 title: Testing run status sensors
 ---
 
+import ScaffoldSensor from '@site/docs/partials/\_ScaffoldSensor.md';
```

Comment:
Same comment here, I'm not sure scaffolding a regular sensor would help with run status sensors, so I think you can remove this. (That may be a feature gap, though.)